homeassistant/helpers/config_validation.py

```diff
@@ -95,6 +95,24 @@ def validate(obj: Dict) -> Dict:
     return validate
 
 
+def has_at_least_one_non_empty_key(*keys: str) -> Callable:
+    """Validate that at least one key exists and is not None."""
+
+    def validate(obj: Dict) -> Dict:
+        """Test keys exist in dict."""
+        if not isinstance(obj, dict):
+            raise vol.Invalid("expected dictionary")
+
+        for k in obj.keys():
```

Comment:
Iterating a dict iterates the keys by default. We can remove `.keys()`.

Reply:
I removed the new function, do you suggest to update the existing functions `has_at_least_one_key`, `has_at_most_one_key` ?

Reply:
That may be ok, since it's a small clean up. Generally I don't recommend mixing new features with clean ups. 🤷‍♂️